<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/*字体*/
QLabel,QToolButton
{font: 12pt &quot;Adobe 黑体 Std R&quot;;}

QRadioButton
{font: 15pt &quot;Adobe 黑体 Std R&quot;;}

QCheckBox,QLineEdit
{font: 13pt &quot;Adobe 黑体 Std R&quot;;}

#btnOK
{font: 20pt &quot;Adobe 黑体 Std R&quot;;}

QComboBox
{font: 10pt &quot;Adobe 黑体 Std R&quot;;}

/*背景图*/
#Register
{background-image: url(:/登录界面的固定图片/Resources/固定/背景2.jpg);}

#btnOK,#btnCHead
{background-image: url(:/登录界面的固定图片/Resources/固定/登录按钮.jpg);}

/*颜色*/
QLineEdit
{background-color:rgb(255, 255, 255,0);color: rgb(255, 255, 255);}

QCheckBox
{color:rgb(255, 255, 255);}

QLabel,QCheckBox,QToolButton,QRadioButton
{color:rgb(255, 255, 255);}
</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>*班级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>真实姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>10位数字</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>662</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineClassName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>例如计算172</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>答案</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAnswer">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密保答案</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>610</y>
     <width>214</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确     定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>*姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>*学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassWord">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>72</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>*密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>密保问题</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbQuestion">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>不设置密保</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>你的真实姓名是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>你的生日是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>你父亲的名字是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>你母亲的名字是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>你最喜欢的明星是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>你最好的朋友是？</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBUser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>普通用户</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBMan">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>560</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
  </widget>
  <widget class="HeadPor" name="frame">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCHead">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择头像</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioSexMan">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioSexWoman">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>*学院</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineInstitute">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>例如：信息</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HeadPor</class>
   <extends>QFrame</extends>
   <header location="global">headpor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../BMS.qrc"/>
 </resources>
 <connections/>
</ui>
